packages/eslint-plugin-khan/test/rules/aphrodite-add-style-variable-name.test.ts

```diff
@@ -0,0 +1,120 @@
+import {RuleTester} from "@typescript-eslint/rule-tester";
+
+import {rules} from "../../src/index";
+
+const ruleTester = new RuleTester({
+    languageOptions: {
+        parserOptions: {
+            ecmaVersion: 6,
+            sourceType: "module",
+            ecmaFeatures: {},
+        },
+    },
+    linterOptions: {
+        // NOTE(kevinb): Avoids 'TypeError: Expected a Boolean' error
+        // when running the tests.
+        reportUnusedDisableDirectives: true,
+    },
+});
+
+const ruleName = "aphrodite-add-style-variable-name";
+const rule = rules[ruleName];
+
+ruleTester.run(ruleName, rule, {
+    valid: [
+        {
+            code: `const StyledDiv = addStyle("div")`,
+        },
+        {
+            code: `const StyledSpan = addStyle("span")`,
+        },
+        {
+            code: `const StyledImg = addStyle("img")`,
+        },
+        {
+            code: `const StyledUl = addStyle("ul")`,
+        },
+        {
+            code: `const StyledOl = addStyle("ol")`,
+        },
+        {
+            code: `const StyledLi = addStyle("li")`,
+        },
+        {
+            code: `const StyledButton = addStyle("button")`,
+        },
+        {
+            code: `const StyledP = addStyle("p")`,
+        },
+        {
+            code: `const StyledSup = addStyle("sup")`,
+        },
```

Comment:
suggestion: It might be worth handling things like `addStyle("foo-bar")` to enable handling of custom HTML components.  I know we don't use any right now, but we might someday in the future or other people/orgs might use this eslint rule (it is open source).  Being pro-active is pretty low cost in this situation.

Reply:
Good idea! Instead of capitalizing the tag name for the expected variable name, I updated it to turn the string into PascalCase. I added a test for this!

Reply:
Awesome. Thank you!